lib/matplotlib/tests/test_transforms.py

```diff
@@ -14,6 +14,340 @@
+                                  [[-2, 0], [-3, 3], [0, 4]])
+
+        r_pi = Affine2D().rotate(np.pi)
+        r180 = Affine2D().rotate_deg(180)
+        assert_array_equal(r_pi.get_matrix(), r180.get_matrix())
+        assert_array_almost_equal(r180.transform(self.single_point), [-1, -1])
+        assert_array_almost_equal(r180.transform(self.multiple_points),
+                                  [[0, -2], [-3, -3], [-4, 0]])
+
+        r_pi_3_2 = Affine2D().rotate(3 * np.pi / 2)
+        r270 = Affine2D().rotate_deg(270)
+        assert_array_equal(r_pi_3_2.get_matrix(), r270.get_matrix())
+        assert_array_almost_equal(r270.transform(self.single_point), [1, -1])
+        assert_array_almost_equal(r270.transform(self.multiple_points),
+                                  [[2, 0], [3, -3], [0, -4]])
+
+        assert_array_equal((r90 + r90).get_matrix(), r180.get_matrix())
+        assert_array_equal((r90 + r180).get_matrix(), r270.get_matrix())
+
+    def test_rotate_around(self):
+        r_pi_2 = Affine2D().rotate_around(*self.pivot, np.pi / 2)
+        r90 = Affine2D().rotate_deg_around(*self.pivot, 90)
+        assert_array_equal(r_pi_2.get_matrix(), r90.get_matrix())
+        assert_array_almost_equal(r90.transform(self.single_point), [1, 1])
+        assert_array_almost_equal(r90.transform(self.multiple_points),
+                                  [[0, 0], [-1, 3], [2, 4]])
+
+        r_pi = Affine2D().rotate_around(*self.pivot, np.pi)
+        r180 = Affine2D().rotate_deg_around(*self.pivot, 180)
+        assert_array_equal(r_pi.get_matrix(), r180.get_matrix())
+        assert_array_almost_equal(r180.transform(self.single_point), [1, 1])
+        assert_array_almost_equal(r180.transform(self.multiple_points),
+                                  [[2, 0], [-1, -1], [-2, 2]])
+
+        r_pi_3_2 = Affine2D().rotate_around(*self.pivot, 3 * np.pi / 2)
+        r270 = Affine2D().rotate_deg_around(*self.pivot, 270)
+        assert_array_equal(r_pi_3_2.get_matrix(), r270.get_matrix())
+        assert_array_almost_equal(r270.transform(self.single_point), [1, 1])
+        assert_array_almost_equal(r270.transform(self.multiple_points),
+                                  [[2, 2], [3, -1], [0, -2]])
+
+        assert_array_almost_equal((r90 + r90).get_matrix(), r180.get_matrix())
+        assert_array_almost_equal((r90 + r180).get_matrix(), r270.get_matrix())
+
+    def test_scale(self):
+        sx = Affine2D().scale(3, 1)
+        sy = Affine2D().scale(1, -2)
+        trans = Affine2D().scale(3, -2)
+        assert_array_equal((sx + sy).get_matrix(), trans.get_matrix())
+        assert_array_equal(trans.transform(self.single_point), [3, -2])
+        assert_array_equal(trans.transform(self.multiple_points),
+                           [[0, -4], [9, -6], [12, 0]])
+
+    def test_skew(self):
+        trans_rad = Affine2D().skew(np.pi / 12, np.pi / 12)
+        trans_deg = Affine2D().skew_deg(15, 15)
```

Comment:
Thinking a bit more, this should probably skew different angles for `x` and `y`.

Reply:
Talked with @ksunden and decided to use a couple of back-calculated values that produce a reasonable set of results.